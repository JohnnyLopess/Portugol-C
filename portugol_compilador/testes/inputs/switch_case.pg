programa{
inicio
inteiro opcao
leia(opcao)
escolha(opcao) {
    caso 1:
        escreva("Opção 1")
    pare
    caso 2:
        escreva("Opção 2")
    pare
    caso contrario:
        escreva("Opção inválida")
}
}