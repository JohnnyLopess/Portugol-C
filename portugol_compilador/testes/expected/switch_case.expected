#include <stdio.h>

int main(){
    int opcao;
    scanf("%d", &opcao);
    switch(opcao) {
    case 1:
        printf("Opção 1");
        break;
    case 2:
        printf("Opção 2");
        break;
    default:
        printf("Opção inválida");
}
    return 0;
}